crates/shrs_core/src/builtin/help.rs

```diff
@@ -53,18 +55,41 @@ impl BuiltinCmd for HelpBuiltin {
                     ctx.out.println(format!("{}: {}", binding, desc))?;
                 }
             },
-            Commands::Plugins => {
-                ctx.out
-                    .println(format!("{} Plugins installed", sh.plugin_metas.len()))?;
-                for meta in sh.plugin_metas.iter() {
-                    ctx.out.println("")?;
+            Commands::Plugin{plugin_name} => {
+                let plg_name = plugin_name.join(" ");
```

Comment:
nit: `plg_name` is a bit unclear imo, would just rename to `plugin_name` and shadow the original variable

Reply:
Done.